pytest-profiling/pytest_profiling.py

```diff
@@ -98,11 +99,14 @@ def pytest_addoption(parser):
                     help="generate profiling graph (using gprof2dot and dot -Tsvg)")
     group.addoption("--pstats-dir", nargs=1,
                     help="configure the dump directory of profile data files")
+    group.addoption("--element-number", action="store", default="20",
+                    help="defines how many elements will display in a result")
```

Comment:
Add `type="int"` here and set the default to `20` instead of the string. This would prevent you having to cast this to an int and potentially blowing up as the profiler is printing the summary.